tests/integration-tests/tests/update/test_update.py

```diff
@@ -385,7 +385,7 @@ def _check_volume(cluster, config, region):
 
 
 @pytest.mark.dimensions("eu-west-1", "c5.xlarge", "alinux2", "awsbatch")
-@pytest.mark.usefixtures("os", "scheduler", "instance")
```

Comment:
We should probably keep it to make sure the dimension in correctly collected in our dashboard